diff --git a/node_modules/onnxruntime-react-native/android/build.gradle b/node_modules/onnxruntime-react-native/android/build.gradle
index 739a6d1..0107201 100644
--- a/node_modules/onnxruntime-react-native/android/build.gradle
+++ b/node_modules/onnxruntime-react-native/android/build.gradle
@@ -228,6 +228,7 @@ def findReactNativeSources() {
 }
 
 dependencies {
+  def ortVersion = "1.24.3"
   //noinspection GradleDynamicVersion
   implementation "com.facebook.react:react-android:"+ REACT_NATIVE_VERSION
   api "org.mockito:mockito-core:2.28.2"
@@ -236,9 +237,9 @@ dependencies {
   implementation "junit:junit:4.12"
 
   if (useQnn) {
-    extractLibs "com.microsoft.onnxruntime:onnxruntime-android-qnn:latest.integration@aar"
+    extractLibs "com.microsoft.onnxruntime:onnxruntime-android-qnn:${ortVersion}@aar"
   } else {
-    extractLibs "com.microsoft.onnxruntime:onnxruntime-android:latest.integration@aar"
+    extractLibs "com.microsoft.onnxruntime:onnxruntime-android:${ortVersion}@aar"
   }
 
   if (VersionNumber.parse(REACT_NATIVE_VERSION) < VersionNumber.parse("0.71")) {
@@ -248,7 +249,7 @@ dependencies {
 
   // By default it will just include onnxruntime full aar package
   if (ortExtensionsEnabled) {
-    implementation "com.microsoft.onnxruntime:onnxruntime-extensions-android:latest.integration@aar"
+    implementation "com.microsoft.onnxruntime:onnxruntime-extensions-android:${ortVersion}@aar"
   }
 }
